src/www/views/servers_view/server_list_item/server_card/index.ts

```diff
@@ -73,6 +73,10 @@ const sharedCSS = css`
     align-items: center;
   }
 
+  .card-metadata-text {
```

Comment:
We're planning on supporting different screen widths and this will break that.

Does `word-break: break-all;` work for your use case?

Reply:
Thank you very much for your feedback, this is my first open source contribution. I completely forgot about the word wrap, it's better to use it here. I made changes and updated the merge request